containers/init/ready/ready.go

```diff
@@ -252,6 +261,48 @@ func WaitForReadyPods(ctx context.Context, ltg LoadTestGetter, pl PodLister, tes
 	return podAddresses, &nodesInfo, nil
 }
 
+func communicateWithEachClient(clientIP string, targets []*pb.Endpoint, isProxied bool) (string, error) {
+	var psmServerTargetOverride string
+	dialTarget := net.JoinHostPort(clientIP, fmt.Sprint(testconfig.ServerUpdatePort))
+	conn, err := grpc.Dial(dialTarget, grpc.WithInsecure())
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+	}
+	defer conn.Close()
+	c := pb.NewTestUpdaterClient(conn)
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
Please raise this 20 to a constant at the top of the file so it can be modified more easily.

Reply:
Missed it, I was using 20 for testing, forgot to update it. Thank you for pointing out!